src/AbstractServer.php

```diff
@@ -126,11 +113,12 @@ protected function _buildSignature(Reflection\AbstractFunction $reflection, $cla
     /**
      * Dispatch method
      *
+     * @param Method\Definition $invokable
+     * @param array $params
```

Comment:
All parameters in DocBlocks can be omitted if there is no more information than in the parameter list of the method itself. The same for the return value.

Reply:
I left it because of @deprecated and scheduled change for 3.0 to remove underscore prefix.

Reply:
I think you misunderstood me. I mean the lines with `@param` and `@return`, these are not needed here.

Reply:
If I understand you right, you suggest removing @param and @return tags which are handled by type declarations? Would you keep @throws tags?
So far I've either had a complete DocBlock or none. Because incomplete DocBlocks do not validate.
![Missing PHPDoc](https://user-images.githubusercontent.com/1815979/81269590-cfb55000-9049-11ea-9699-7b72dad676ef.jpg)

Reply:
Search for "Redundant PHPDoc comment" (Inspections) in the preferences of PhpStorm this should solve the problem.

Reply:
Got it, thank you.

Reply:
I thank you for your work. 👍